src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -4,6 +4,13 @@ import { ItemData } from '../../services/item-datasource.service';
 import { GetItemChildrenService } from '../../http-services/get-item-children.service';
 import { Subscription } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { ItemType } from '../../../../shared/helpers/item-type';
+
+export interface NewItem {
```

Comment:
It is the duty of the emitter to define what he sends, not the listening component! 

What the creating component knows is just the title and the type, I would just return these two `{ title: string, type: ItemType }`. It is not the job of the add-content component to decide of the language a priori.

Reply:
Done